js/data/boards.js

```diff
@@ -165,5 +165,18 @@ var BOARDS =[
             {name:'Annabel van den Broek',role:'act'}],
         img: 'images/besturen/13ebestuur.jpg'
 
+    },
+    {
+        prefixDutch: "Veertiende",
+        prefixEng: "Fourteenth",
+        durationDutch: "September 2019 - September 2020",
```

Comment:
Names of months should start with a lower-case letter in Dutch... but I see it's done like this in this whole list now :(